x-pack/test/functional/apps/user_profiles/user_profiles.ts

```diff
@@ -118,14 +118,12 @@ export default ({ getPageObjects }: FtrProviderContext) => {
           shouldUseHashForSubUrl: false,
         });
 
-        let advancedSetting = await pageObjects.settings.getAdvancedSettingCheckbox(
-          'theme:darkMode'
-        );
-        expect(advancedSetting).to.be(null);
+        let advancedSetting = await pageObjects.settings.getAdvancedSettings('theme:darkMode');
+        expect(advancedSetting).to.be('disabled');
 
-        await pageObjects.settings.toggleAdvancedSettingCheckbox('theme:darkMode', true);
-        advancedSetting = await pageObjects.settings.getAdvancedSettingCheckbox('theme:darkMode');
-        expect(advancedSetting).to.be('true');
+        await pageObjects.settings.setAdvancedSettingsSelect('theme:darkMode', 'enabled');
+        advancedSetting = await pageObjects.settings.getAdvancedSettings('theme:darkMode');
+        expect(advancedSetting).to.be('enabled');
```

Comment:
The `theme:darkMode` setting changed from a boolean toggle to a select - had to adapt the test